### PR TITLE
game.watch.impress.co.jp

### LINKED ISSUE #103845: game.watch.impress.co.jp
### Issue URL (Ads)
 [https://game.watch.impress.co.jp/docs/news/1375452.html](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fgame.watch.impress.co.jp%2Fdocs%2Fnews%2F1375452.html)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard/AdGuard Pro for iOS v4.3.0
Browser: | Safari
DNS filtering: | disabled
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances<br/><br/><b>Other:</b><br/>AdGuard Experimental<br/><br/><b>Language-specific:</b><br/>AdGuard Russian, <br/>AdGuard Japanese, <br/>AdGuard Chinese

### PR DESCRIPTION
**Pull request by SKEIDs**
Fix #103845

### CODE REVIEW

File: MobileFilter/sections/specific_web.txt [hunk 1]
```diff
@@ -28,6 +28,8 @@ theinventory.com,avclub.com,clickhole.com,deadspin.com,gizmodo.com,jalopnik.com,
 cartoonpornvideos.com,m.economictimes.com,beethovenmaster.com,m.mistrzowie.org,m.ura.news,siirtpress.com,up.ymgt.xyz,google.de,www-bento-de.cdn.ampproject.org,m-economictimes-com.cdn.ampproject.org,m.winfuture.de##.ads
 !
 news24.jp##.adText > ul
+game.watch.impress.co.jp##aside[class^="ad"]
```
Comment (Yuki2718) [inline MobileFilter/sections/specific_web.txt:31]: This rule has false positive on desktop site (viewable on mobile too). Please change to `watch.impress.co.jp##.ad.movie`.

File: MobileFilter/sections/specific_web.txt [hunk 2]
```diff
@@ -28,6 +28,8 @@ theinventory.com,avclub.com,clickhole.com,deadspin.com,gizmodo.com,jalopnik.com,
...
+game.watch.impress.co.jp###sp-floating-layer
```
Comment (Yuki2718) [inline MobileFilter/sections/specific_web.txt:32]: Please drop `game.` subdomain.

**Review by Yuki2718 — CHANGES_REQUESTED** [review]
Add `watch.impress.co.jp` to `#?#li > ._popIn_recommend_article_ad:upward(1)` rule in JPF > css_extended and also add `watch.impress.co.jp##.extension > aside.special` to Annoyances > self-promo.

**Comment by SKEIDs** [comment]
Fixed all

**Review by Yuki2718 — APPROVED** [review]

**Review by zloyden — APPROVED** [review]

**Pull request merged**